src/SelectableGroup.tsx

```diff
@@ -490,12 +493,12 @@ class SelectableGroup extends Component<TSelectableGroupProps> {
       const offsetData = getBoundsForNode(this.selectableGroup!)
```

Comment:
`this.selectableGroup` return only one client rect so it might be simplified via destructuring:

```ts
 const [offsetData] = getBoundsForNode(this.selectableGroup!)
```